doc_source/guide_configuration.rst

```diff
@@ -625,6 +625,18 @@ You can use the ``HTTP_PROXY`` environment variable to configure an "http"
 protocol-specific proxy, and the ``HTTPS_PROXY`` environment variable to
 configure an "https" specific proxy.
 
+.. _http_expect:
+
+expect

```

Comment:
There should probably be a warning that overriding default behavior may impact service operations that need this set a certain way.